lib/charms/kubeflow_dashboard/v0/kubeflow_dashboard_links.py

```diff
@@ -123,10 +123,10 @@ class KubeflowDashboardLinksProvider(Object):
     on = KubeflowDashboardLinksEvents()
 
     def __init__(
-        self,
-        charm: CharmBase,
-        relation_name: str,
-        refresh_event: Optional[Union[BoundEvent, List[BoundEvent]]] = None,
+            self,
```

Comment:
Extra spaces.

Reply:
yep!  copy and paste error I guess?  good catch